app/services/ccms/attribute_value_generator.rb

```diff
@@ -179,6 +179,24 @@ def lead_proceeding_category_of_law_code(_options)
       lead_proceeding_type.ccms_category_law_code
     end
 
+    def ccms_equivalent_prospects_of_success(_options) # rubocop:disable Metrics/MethodLength
+      check_value = @legal_aid_application.merits_assessment.success_prospect
+      case check_value
+      when 'likely'
+        'Good'
+      when 'marginal'
+        'Marginal'
+      when 'poor'
+        'Poor'
+      when 'borderline'
+        'Borderline'
+      when 'uncertain'
+        'Uncertain'
+      else
+        "Could not find match for #{check_value}"
+      end
```

Comment:
A `case` statement is a little over-kill when each of the options simply return a small string. I'd suggest:

```ruby
PROSPECTS_OF_SUCCESS = {
  likely: 'Good',
  marginal: 'Marginal',
  poor: 'Poor',
  borderline: 'Borderline'
  uncertain: 'Uncertain'
}.freeze

attr_reader :legal_aid_application
delegate :merits_assessment, to: :legal_aid_application

def ccms_equivalent_prospects_of_success(_options)
  PROSPECTS_OF_SUCCESS[merits_assessment.success_prospect.to_sym] || "Could not find match for #{check_value}"
end
```